interfaces/builtin/tmpfs_mount.go

```diff
@@ -0,0 +1,78 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const tmpfsMountSummary = `allows mounting and unmounting tmpfs filesystems`
+
+const tmpfsMountBaseDeclarationSlots = `
```

Comment:
this matches the cifs-mount declaration which I think is reasonable for this interface - we don't auto-connect by default, but we also allow folks to just declare a plug in their snap without intervention (this is opposed to something like k8s-support, where you need store approval just to upload a snap which plugs it)